ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -279,6 +280,19 @@ public void setBorderStyle(@Nullable String style) {
     getOrCreateReactViewBackground().setBorderStyle(style);
   }
 
+  public void setPreventClipping(boolean preventClipping) {
+    mPreventClipping = preventClipping;
+
+    // TODO(apkumar)
```

Comment:
Is this TODO blocking for this merge? Or more like something you plan to fix at some future date?

Reply:
I don't think it's blocking merge, it still works as expected as far as I can tell. It would just be a bit more accurate.